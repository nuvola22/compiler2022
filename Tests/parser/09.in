procedure (var b: Integer; const c: Real; d: String);
begin
    writeln(b, c, d);
end;

begin end.