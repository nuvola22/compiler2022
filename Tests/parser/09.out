(1, 11), Id expected
